docs/en/operations/settings/settings.md

```diff
@@ -1397,6 +1397,64 @@ Possible values:
 
 Default value: 16.
 
+## transform_null_in {#transform_null_in}
+
+Enables or disables NULL-to-zero transformation in the `IN` operator.
+
+By default, the `IN` operator processes `NULL` as 0.  
+If `transform_null_in` is enabled, the `IN` operator processes `NULL` as [NULL](../../sql-reference/syntax.md#null-literal). 
```

Comment:
Let's explicitly explain how this works when the setting is enabled. In normal binary operators, two NULLs are not supposed to be equal (treated as "unknown"), while in the example below `NULL IN (...NULL)` is true.